src/sensor/settings.py

```diff
@@ -90,7 +93,7 @@
 
 # See /env.template
 if not IN_DOCKER or RUNNING_TESTS:
-    SECRET_KEY = "!j1&*$wnrkrtc-74cc7_^#n6r3om$6s#!fy=zkd_xp(gkikl+8"  # TODO not sure why this is set here
+    SECRET_KEY = "!j1&*$wnrkrtc-74cc7_^#n6r3om$6s#!fy=zkd_xp(gkikl+8"
```

Comment:
I'm guessing this is a not-so-secret key used as a placeholder for running tests- just commenting to make sure that this should be included here.